Editor/MarkerEditor.cs

```diff
@@ -622,9 +622,9 @@ private void CheckRequirements()
 				warnings.Add("There is no avatar descriptor on this GameObject. Please move this script onto your avatar, or create an avatar descriptor here.");
 			else
 			{
-				if (descriptor.expressionParameters != null && descriptor.expressionParameters.CalcTotalCost() > (128 - bitCount))
+				if (descriptor.expressionParameters != null && descriptor.expressionParameters.CalcTotalCost() > (256 - bitCount))

```

Comment:
It was an error on our side to hardcode the bits total cost here, it should not be hardcoded and use `VRCExpressionParameters.MAX_PARAMETER` instead